apollo-federation/src/sources/connect/expand/tests/schemas/expand/interface-object.yaml

```diff
@@ -0,0 +1,84 @@
+# rover supergraph compose --config apollo-router/src/plugins/connectors/testdata/interface-object.yaml > apollo-router/src/plugins/connectors/testdata/interface-object.graphql
+# the result is manually edited because 2.10 disallows interface object
+federation_version: =2.10.0-local.3 # switch to .3 or .4 when composition support for interfaceObject lands
```

Comment:
This version is actually overridden by [a script](https://github.com/apollographql/router/blob/0def3e6c72c48afadb94a0146df828c09bbdf6a5/apollo-federation/src/sources/connect/expand/tests/schemas/regenerate.sh) now—At least I _think_ the env var overrides the suergraph.yaml version. So maybe just a comment in case someone runs that and gets an error?

```suggestion
# Requires composition >= 2.10.0-preview.3
```